rskj-core/src/main/java/co/rsk/core/ReversibleTransactionExecutor.java

```diff
@@ -81,15 +83,10 @@ public ProgramResult executeTransaction(
         );
 
         TransactionExecutor executor = new TransactionExecutor(
-                config,
-                tx,
-                0,
-                coinbase,
-                repository,
-                blockStore,
-                receiptStore,
-                programInvokeFactory,
-                executionBlock
+                tx, 0, coinbase, repository, blockStore, receiptStore,
+                programInvokeFactory, executionBlock, new EthereumListenerAdapter(), 0, config.getVmConfig(),
+                config.getBlockchainConfig(), config.playVM(), config.isRemascEnabled(), config.vmTrace(), new PrecompiledContracts(config),
+                config.databaseDir(), config.vmTraceDir(), config.vmTraceCompressed()
```

Comment:
Here you could rework config.getVmConfig() object to have all vm config data related and reduce parameters

Reply:
Maybee it's time
```  //TODO: REMOVE THIS WHEN THE LocalBLockTests starts working with REMASC
    public boolean isRemascEnabled() {
        return remascEnabled;
    }```